dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/ConnectionManagerServiceImpl.java

```diff
@@ -619,6 +581,15 @@ else if (value instanceof Number)
                     throw resX;
             }
 
+        //Check if immediate is supported.  If not default value?
```

Comment:
This should probably be `//Check if immediate is supported.  If not throw exception`